src/azure-cli/azure/cli/command_modules/appconfig/_utils.py

```diff
@@ -254,3 +261,73 @@ def parse_tags_to_dict(tags):
                 tags_dict[tag_key] = tag_value
         return tags_dict
     return tags
+
+
+def strip_json_comments(json_string):
+    current_index = 0
+    length = len(json_string)
+    result = []
+
+    def __is_escaped(json_string, char_index):
+        backslash_count = 0
+        index = char_index - 1
+        while index >= 0 and json_string[index] == BACKSLASH:
+            backslash_count += 1
+            index -= 1
+
+        return backslash_count % 2 == 1
+
+    def __find_next_newline_index(json_string, start_index):
+        index = start_index
+
+        while index < length:
+            if json_string[index] == NEW_LINE:
+                break
+
+            index += 1
+
+        return index
+
+    def __find_next_double_quote_index(json_string, start_index):
```

Comment:
`start_index` seems like an odd name. And unless I'm missing something it can just be index.

Reply:
The return value is already an index. Passing in another variable called "index", to me, is more confusing